src/vs/workbench/services/policies/common/accountPolicyService.ts

```diff
@@ -0,0 +1,41 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IStringDictionary } from '../../../../base/common/collections.js';
+import { ILogService } from '../../../../platform/log/common/log.js';
+import { AbstractPolicyService, IPolicyService, PolicyDefinition } from '../../../../platform/policy/common/policy.js';
+import { DefaultAccountService, IDefaultAccountService } from '../../accounts/common/defaultAccount.js';
+
+export class AccountPolicyService extends AbstractPolicyService implements IPolicyService {
+	// private editorPreviewFeaturesEnabled: boolean = false;
+	constructor(
+		@ILogService private readonly logService: ILogService,
+		@IDefaultAccountService private readonly defaultAccountService: DefaultAccountService
+	) {
+		super();
+		this._register(this.defaultAccountService.onDidChangeDefaultAccount((account) => {
+			this.logService.info(`account?=${account?.sessionId} previewFeatures=${account?.editor_preview_features_enabled}`);
+			// this.editorPreviewFeaturesEnabled = !!account?.editor_preview_features_enabled;
+		}));
+	}
+
+	protected async _updatePolicyDefinitions(policyDefinitions: IStringDictionary<PolicyDefinition>): Promise<void> {
+		this.logService.info(`AccountPolicyService#_updatePolicyDefinitions: Got ${Object.keys(policyDefinitions).length} policy definitions`);
+		// if (this.editorPreviewFeaturesEnabled) {
+		// 	return;
+		// }
+
+		const update: string[] = [];
+		for (const key in policyDefinitions) {
+			this.logService.info(`AccountPolicyService#_updatePolicyDefinitions: key=${key} policyDefinition=${JSON.stringify(policyDefinitions[key])}`);
+			if (key.startsWith('Copilot')) {
```

Comment:
A quick way to discern which policy this  'multiplexed' policy service should handle.

Another option is a flag on `IPolicy` that does filtering earlier.  Also open to other ideas that could be more scalable.

Reply:
Another option would be to tag policy definition with `preview` flag:

```ts
export interface IPolicy {

	/**
	 * The policy name.
	 */
	readonly name: PolicyName;

	/**
	 * The Code version in which this policy was introduced.
	*/
	readonly minimumVersion: `${number}.${number}`;

	/**
	 * The policy description (optional).
	 */
	readonly description?: string;

	/**
	 * Is preview feature
	 */
	readonly previewFeature?: boolean;

	/**
	 * Default value when enabled. Default is `false`.
	 */
	readonly defaultValue?: any;
}
```

If policy is for a preview feature and preview features are disabled for the account, then set the value of the policy to the default value defined in the policy or `false`

Reply:
Perhaps slightly more generic?

```ts
export interface IPolicy {
       // ...
       // ... 
 
	/**
	 * The policy value is fixed to `defaultValue` and cannot be changed externally.
	 */
	readonly fixed?: boolean;

	/**
	 * Default value when enabled. Default is `false`.
	 */
	readonly defaultValue?: any;
}
```

Reply:
or `readonly protected?: boolean`